src/app/layouts/landing/landing.tsx

```diff
@@ -8,27 +8,34 @@ import useMainClassContext, {
 import useLanguageContext from '~/contexts/language';
 import ReactModal from 'react-modal';
 import cn from 'classnames';
-import { LinkFields } from '../../pages/flex-page/components/Link';
+import {LinkFields} from '../../pages/flex-page/components/Link';
 import './landing.scss';
 
 type Props = {
+    showGive?: boolean;
```

Comment:
we're gonna want this to arrive via `data.layout[0]?.value.showGive`

Reply:
I have updated the code & test accordingly.